src/components/atoms/Checkbox/index.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import classNames from 'classnames'
+
+import Input from '_atoms/Input'
+
+import styles from './styles.css'
+
+const Checkbox = props => {
+  const { className, isChecked } = props
+  const inputClasses = classNames({
```

Comment:
When using classnames with fixed and dynamic fields you can use it like this:
`classNames(className, styles.input, { [styles.checked]: isChecked })` 
No need to add the `true` to the fixed items.